index.html

```diff
@@ -28,19 +28,21 @@ <h2>Code Journal</h2>
       <div class="row">
           <form id="form" class="column-full">
             <h1>New Entry</h1>
-            <div class="display-flex">
-              <img id="main-image" src="images/placeholder-image-square.jpg">
-              <div class="column-full display-inline">
+            <div class="row">
+              <img id="main-image" class="column-full" src="images/placeholder-image-square.jpg">
```

Comment:
Shouldn't this be `column-half`? The image only takes up half of the row on desktop.